slack_sdk/web/client.py

```diff
@@ -1022,6 +1022,25 @@ def admin_users_session_clearSettings(
             kwargs.update({"user_ids": user_ids})
         return self.api_call("admin.users.session.clearSettings", params=kwargs)
 
+    def admin_users_unsupportedVersions_export(
+        self,
+        *,
+        date_end_of_support: Optional[Union[str, int]] = None,
```

Comment:
The value should be an integer but, as with other methods in this SDK, both int and str should be accepted here.